### DISCUSSION TITLE
Feature: allow management of notification from header

### DISCUSSION THREAD

**Discussion by DarkWanderer** [community/community#147100] [Mobile]
In GitHub mobile app, there is a function to browse notifications. After I am done with notification, I am taken back to Notifications screen and have to remember which one I just browsed to dismiss. It gets tedious if you go over dozens notifications at once

Suggestion: use currently wasted screen estate to place "Done" and "Unsubscribe" buttons, similar to web version, which would allow to go through notifications faster

![Screenshot_20241214-173328.png](https://github.com/user-attachments/assets/5cfdcc75-9195-46b0-8766-2c3ca534eebc)

**Comment by eliperkins** [comment]
Hey @DarkWanderer , thanks for this feedback! Managing notifications is definitely a top priority for GitHub Mobile, so making it easy to triage notifications right from the header of the issue/PR/discussion screen definitely makes sense.

I've added this feature request to our team's internal tracking project. I'll let you know when we've prioritized this!

Thanks for helping improve GitHub Mobile. 💪
